components/Common/OramaSearchBar/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { create, load } from '@orama/orama';
+import { useTheme } from 'next-themes';
+import { useCallback, useMemo, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { OramaLogoDark, OramaLogoLight } from './OramaLogo';
+import styles from './index.module.scss';
+import { identityDocumentMapper, performSearch } from './search';
+import SearchBar from '../SearchBar';
+
+import type { FC } from 'react';
+import type { DocumentMapper, OramaSearchBarProps } from './search';
+import type { SearchFunction } from '../../../types';
+
+const Footer: FC = () => {
+  const { resolvedTheme: theme } = useTheme();
+
+  const isDark = useMemo(() => theme === 'dark', [theme]);
```

Comment:
No need of useMemo here, since there is no heavy computation stuff
```suggestion
  const isDark = theme === 'dark';
```

Reply:
👍